siriushlafac/si_ap_coupling_meas/main.py

```diff
@@ -0,0 +1,318 @@
+"""Main module of the Application Interface."""
+import os as _os
+import logging as _log
+from threading import Thread
+import pathlib as _pathlib
+
+import numpy as np
+import matplotlib.pyplot as mplt
+import matplotlib.gridspec as mgs
+from matplotlib import rcParams
+
+from qtpy.QtCore import Qt
+from qtpy.QtGui import QDoubleValidator
+from qtpy.QtWidgets import QWidget, QPushButton, QGridLayout, QSpinBox, \
+    QLabel, QGroupBox, QLineEdit, QComboBox, QHBoxLayout, QFileDialog, \
+    QVBoxLayout, QMessageBox
+
+import qtawesome as qta
+
+# from pydm.widgets.logdisplay import PyDMLogDisplay
+
+from siriushla import util
+from siriushla.widgets import MatplotlibWidget, SiriusMainWindow, \
+    SiriusLogDisplay
+
+from apsuite.commissioning_scripts.meas_coupling_tune import MeasCoupling
+
+rcParams.update({
+    'font.size': 12, 'axes.grid': True, 'grid.linestyle': '--',
+    'grid.alpha': 0.5})
+
+
+class SICoupMeasWindow(SiriusMainWindow):
+    """."""
+    EXT = 'pickle'
+    EXT_FLT = f'Pickle Files (*.{EXT:s})'
+    DEFAULT_DIR = _pathlib.Path.home().as_posix()
+    DEFAULT_DIR += _os.path.sep + _os.path.join(
+        'mounts', 'screens-iocs', 'data_by_day')
+    print(DEFAULT_DIR)
+
+    def __init__(self, parent=None):
+        """."""
+        super().__init__(parent=parent)
+        self.meas_coup = MeasCoupling()
+        self._last_dir = self.DEFAULT_DIR
+
+        self.setupui()
+        self.setObjectName('SIApp')
+        color = util.get_appropriate_color('SI')
+        icon = qta.icon('mdi.notebook', 'mdi.pulse', options=[
+            dict(scale_factor=0.4, color=color, offset=(0.1, -0.3)),
+            dict(scale_factor=1, color=color, offset=(0, 0.0))])
+        self.setWindowIcon(icon)
+
+    def setupui(self):
+        """."""
+        self.setWindowModality(Qt.WindowModal)
+        self.setWindowTitle("SI - tune Measurement")
```

Comment:
I think that is "SI - Coupling Measurement" here too

Reply:
done!